core/src/apps/management/apply_settings.py

```diff
@@ -11,6 +11,16 @@
     from trezor.messages.ApplySettings import ApplySettings
 
 
+def humanize_time(milliseconds: int) -> str:
+    units = (("hour", 60 * 60 * 1000), ("minute", 60 * 1000), ("second", 1000))
+    for (u, divisor) in units:
```

Comment:
This is very nit-picky but I prefer `unit` rather then `u`. It makes the code slightly more readable imho.

Reply:
Done 8516e7a.